### ISSUE TITLE
Versions are not picked up from parent poms

### ISSUE THREAD

**Issue by ashwinraghav** [jaredsburrows/gradle-license-plugin#72]
Some projects like `okHttp` seems to not codify versions directly, but in the parent pom files that have a 1:1 mapping with each release.
I can't tell if this is in accordance to the pom spec. But the maven plugin for gradle itself seems to honor this codification.

```xml
<parent>
    <groupId>com.squareup.okhttp3</groupId>
    <artifactId>parent</artifactId>
    <version>3.12.1</version>
  </parent>
```

The problem I see if that the license plugin has versions set to null, which is frankly quite reasonable.
```json
{
        "project": "OkHttp",
        "description": null,
        "version": null
}
```

Could we have this pickup the version from the parent pom instead ?

**Comment by jaredsburrows** [comment]
This feature was done here: https://github.com/jaredsburrows/gradle-license-plugin/pull/27. It also has tests. What version are you using?

**Comment by jaredsburrows** [comment]
Closed by #88
